precode.go

```diff
@@ -39,14 +40,84 @@ var tasks = map[string]Task{
 	},
 }
 
-// Ниже напишите обработчики для каждого эндпоинта
-// ...
+func getTasks(w http.ResponseWriter, r *http.Request) {
+	var tasksSlice []Task
+
+	for _, task := range tasks {
+		tasksSlice = append(tasksSlice, task)
+	}
+
+	resp, err := json.Marshal(tasksSlice)
```

Comment:
Можно не создавать доп слайс (если он просто дублирует основной)
```suggestion
	resp, err := json.Marshal(tasks)
```